### PR TITLE
Add Orthographic View Toggle

### PR DESCRIPTION
**Pull request by lknknm**
As suggested on the Issue #885, this PR has the intention to continue developing the camera hotkeys feature.
This will implement the Ortographic View bound to the `5` key of the keyboard/numeric keyboard, using the `vtkCamera::ParallelProjectionOn/Off()` methods.

### CODE REVIEW

File: library/private/camera_impl.h [hunk 1]
```diff
@@ -57,6 +57,7 @@ class camera_impl : public camera
   camera& yaw(angle_deg_t angle) override;
   camera& elevation(angle_deg_t angle) override;
   camera& pitch(angle_deg_t angle) override;
+  camera& ortoview() override;
```
Comment (Meakk) [inline library/private/camera_impl.h:60]: English is not my native language but it seems like a `h` is missing.  
Please, someone confirm?
image

File: library/src/camera_impl.cxx [hunk 2]
```diff
@@ -214,6 +214,17 @@ camera& camera_impl::pitch(angle_deg_t angle)
...
 }
 
+//----------------------------------------------------------------------------
+camera& camera_impl::ortoview()
+{
+  vtkCamera* cam = this->GetVTKCamera();
+  if (cam->GetParallelProjection() == 0)
+    cam->ParallelProjectionOn();
+  else
+    cam->ParallelProjectionOff();
```
Comment (Meakk) [inline library/src/camera_impl.cxx:224]:
```suggestion
  cam->SetParallelProjection(!cam->GetParallelProjection());
```

File: library/src/camera_impl.cxx [hunk 3]
```diff
@@ -214,6 +214,17 @@ camera& camera_impl::pitch(angle_deg_t angle)
   return *this;
 }
 
+//----------------------------------------------------------------------------
+camera& camera_impl::ortoview()
```
Comment (Meakk) [inline library/src/camera_impl.cxx:218]: Maybe a better name would be `toggleOrthographic()` or `togglePerspective()` to clearly understand calling it again revert to the original state.

**Review by Meakk — CHANGES_REQUESTED** [review]
Please add the Python bindings. You don't have to add the Java and JS bindings tho, except if you really want :)  
You also need tests in both C++ and Python since you are extending the public API.  
An interactive test (with `add_test`) is also required.

**Comment by mwestphal** [comment]
superseeded by https://github.com/f3d-app/f3d/pull/1235

**Pull request closed without merging**